user_test.go

```diff
@@ -103,6 +103,40 @@ func TestListUser(t *testing.T) {
 	autopilot.Equals(t, ol.UserRoleAdmin, result[2].Role)
 }
 
+func TestListUserOmitDeactivated(t *testing.T) {
+	// Arrange
+	testRequestOne := autopilot.NewTestRequest(
+		`query UserList($after:String!$filter:[UsersFilterInput!]$first:Int!){account{users(after: $after, first: $first, filter: $filter){nodes{id,email,htmlUrl,name,role},{{ template "pagination_request" }},totalCount}}}`,
+		`{ "after": "", "filter": [ {"key": "deactivated_at", "type": "equals"} ], "first": 100 }`,
+		`{ "data": { "account": { "users": { "nodes": [ {{ template "user_1" }}, {{ template "user_2" }} ], {{ template "pagination_initial_pageInfo_response" }}, "totalCount": 2 }}}}`,
+	)
+	testRequestTwo := autopilot.NewTestRequest(
+		`query UserList($after:String!$filter:[UsersFilterInput!]$first:Int!){account{users(after: $after, first: $first, filter: $filter){nodes{id,email,htmlUrl,name,role},{{ template "pagination_request" }},totalCount}}}`,
+		`{ "after": "OA", "filter": [ {"key": "deactivated_at", "type": "equals"} ], "first": 100 }`,
+		`{ "data": { "account": { "users": { "nodes": [], {{ template "pagination_second_pageInfo_response" }}, "totalCount": 0 }}}}`,
+	)
+	requests := []autopilot.TestRequest{testRequestOne, testRequestTwo}
+
+	client := BestTestClient(t, "user/list_omit_deactivated", requests...)
+	// Act
+	payloadVars := client.InitialPageVariablesPointer()
+	(*payloadVars)["filter"] = &[]ol.UsersFilterInput{
+		{
+			Key:  ol.UsersFilterEnumDeactivatedAt,
+			Type: ol.RefOf(ol.BasicTypeEnumEquals),
+		},
+	}
```

Comment:
This is SUPER ugly and unfriendly to the end user.  Any way we can make this customer facing interface better?  Maybe functional options passed into `ListUsers`?

IE
```
response, err := client.ListUsers(nil, ol.WithoutDeactivedUsers())
```